acoustic_msgs/msg/PingInfo.msg

```diff
@@ -0,0 +1,13 @@
+# Center frequency of sonar in Hz
+# Set to 0 if unavailable
+float32 frequency
+
+# Speed of sound (m/s) used to calculate ranges;
+# Set to 0 if unavailable
+float32 sound_speed
+
+# Sonar reported -3db beamwidths
+# May be empty if not reported
+# reported in radians
+float32[] tx_beamwidths
```

Comment:
@CaptKrasno -- are you willing to add a note describing these in a bit more detail? e.g. a roboticist wanting to get started using these may not know how transmit/receive beamforming is performed, and thus won't know which axes these correspond to.